public/modules/dashboards/views/fbf.client.view.html

```diff
@@ -800,4 +829,84 @@ <h2 class="h5 modal-title">{{ "warning" | translate }}</h2>
     </div>
   </div>
   <!-- end: IE message -->
+
+  <!-- start: Mailchimp modal -->
+  <div id="MailChimpModal" tabindex="-1" class="modal" role="dialog">
+    <div class="modal-dialog">
+      <!-- Modal content-->
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal">
+            <i class="fa fa-times-circle-o" aria-hidden="true"></i>
+          </button>
+          <h2 class="h5 modal-title">
+            {{ "Sign up to mailing list" | translate }}
+          </h2>
+        </div>
+        <div class="modal-body">
+          <link
+            href="//cdn-images.mailchimp.com/embedcode/slim-10_7.css"
```

Comment:
I don't think we would need all the external styles loaded from MailChimp.
We could re-use the styles for form-fields we already have from Bootstrap.